### PR TITLE
Booking portal/payment link

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/store/bookingPortal/index.js [hunk 1]
```diff
@@ -17,10 +19,17 @@ const mutations = {
         state.hasError = true;
         state.errorMessage = errorMessage;
     },
+    'set-payment-details'(state, paymentDetails) {
```
Comment (arrogant-ant) [inline src/store/bookingPortal/index.js:22]: add new line before every mutation and functions.

File: src/store/bookingPortal/index.js [hunk 2]
```diff
@@ -29,6 +38,21 @@ const actions = {
         } else if (res.DisplayMsg) {
             commit('set-error', res.DisplayMsg + ' ( ' + res.ErrorMsg + ' )');
         }
+        commit('set-loading', false);
+    },
+    async getPaymentLink({ dispatch, commit }, reqBody) {
```
Comment (arrogant-ant) [inline src/store/bookingPortal/index.js:43]: add new  line before methods

File: src/store/bookingPortal/index.js [hunk 3]
```diff
@@ -29,6 +38,21 @@ const actions = {
...
+        commit('set-loading', true);
+        const res = await mayaClient.post(
+            '/payment/generate-payment-link',
+            reqBody,
+        );
+        if (res) {
```
Comment (arrogant-ant) [inline src/store/bookingPortal/index.js:49]: if there is an error maya will still send res with DisplayMsg but without any paymentDetails
So this condition will still evaluate true
Else condition will never be evaluated

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/store/bookingPortal/index.js (hunk 2 above)
Comment (alias01) [reply src/store/bookingPortal/index.js:43]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**